paddlenlp/trainer/trainer.py

```diff
@@ -1576,6 +1585,9 @@ def get_expected_keys(inputs, keys):
                     offload=cpu_offload,
                     **extra_kwargs,
                 )
+                if level == "os_g":
+                    model._set_reduce_overlap(True)
```

Comment:
Add a flag to control whether using overlap or not? There are some constraints for the overlap, such as the `logging_step` should bigger than 1 for broadcast overlap.

Reply:
Done